pkg/kubectl/cmd/create.go

```diff
@@ -201,8 +201,10 @@ func NameFromCommandArgs(cmd *cobra.Command, args []string) (string, error) {
 
 // CreateSubcommandOptions is an options struct to support create subcommands
 type CreateSubcommandOptions struct {
-	// Name of resource being created
+	// Name of resource instance being created (ex: my-pod, my-namespace)
 	Name string
+	// Resource being created (ex: pod, namespace)
+	Resource string
```

Comment:
This PR only affects CreateSubresourceCommands where user only specifies the resource and we want to figure out the group using the mapper.

Example command:
`kubectl create namespace my-namespace`

We have such create sub commands only for namespace, configmap, secrets and service account.